starboard/shared/stub/string_format.cc

```diff
@@ -14,9 +14,11 @@
 
 #include "starboard/common/string.h"
 
+#if SB_API_VERSION <= 15
```

Comment:
The conversion is to use #if SB_API_VERSION < 16. Please update here and the other places.